flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/TestUtils.java

```diff
@@ -137,6 +137,36 @@ public static FlinkDeployment buildApplicationCluster(FlinkVersion version) {
         return buildApplicationCluster(TEST_DEPLOYMENT_NAME, TEST_NAMESPACE, version);
     }
 
+    public static FlinkDeployment buildApplicationCluster(
```

Comment:
If this is only used in one test class let's keep it there otherwise it gets a bit confusing

Reply:
Moved back to test.